modules/core/src/main/java/org/apache/fluo/core/impl/TransactionImpl.java

```diff
@@ -1571,6 +1579,20 @@ private void beginCommitAsyncTest(CommitData cd) {
   }
 
   public SnapshotScanner newSnapshotScanner(Span span, Collection<Column> columns) {
-    return new SnapshotScanner(env, new SnapshotScanner.Opts(span, columns, false), startTs, stats);
+    return newSnapshotScanner(span, columns, env.getAuthorizations());
```

Comment:
Do we want to use env.getAuthorizations()? What about auths set on the snapshot?

Reply:
Looking into to it, this whole method can be removed.   Just pushed a change to my branch with that update.